### PR TITLE
games-engines/qtads: bump to 3.1.0, EAPI 8

### PR DESCRIPTION
**Pull request by realnc**
`qtads-3.0-fluidsynth.patch` was merged into 3.1 so not needed anymore. Also bumped to EAPI 8.

Closes: https://bugs.gentoo.org/813417
Package-Manager: Portage-3.0.23, Repoman-3.0.3
Signed-off-by: Nikos Chantziaras <realnc@gmail.com>

### CODE REVIEW

File: games-engines/qtads/qtads-3.1.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+		media-sound/fluidsynth:0=
+		media-sound/mpg123
+	)
+"
+RDEPEND=${DEPEND}
+
+src_configure() {
+	eqmake5 \
+		PREFIX="${EPREFIX}/usr" \
+		$(usex !sound CONFIG+=disable-audio '') \
```
Comment (ionenwks) [inline games-engines/qtads/qtads-3.1.0.ebuild:37]: Trivial but should be updated for EAPI-8 if bumping
```suggestion
		$(usev !sound CONFIG+=disable-audio) \
```

File: games-engines/qtads/qtads-3.1.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit qmake-utils xdg
```
Comment (ionenwks) [inline games-engines/qtads/qtads-3.1.0.ebuild:5]: Typically keep a blank line after EAPI

File: games-engines/qtads/qtads-3.1.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+	dev-qt/qtwidgets:5
+	sound? (
+		media-libs/libsdl2[sound]
+		media-libs/libsndfile
+		media-libs/libvorbis
+		media-sound/fluidsynth:0=
+		media-sound/mpg123
+	)
+"
+RDEPEND=${DEPEND}
```
Comment (ionenwks) [inline games-engines/qtads/qtads-3.1.0.ebuild:32]: In global scope it's customary to always use quotes for everything but EAPI=

**Review by ionenwks — COMMENTED** [review]
Thanks, not seeing issues here beside some trivial nitpicks if you don't mind.

**Comment by realnc** [comment]
@ionenwks Thanks for the suggestions. They're now applied.

**Review by ionenwks — APPROVED** [review]
Thanks!

**Pull request closed without merging**